CHANGELOG.md

```diff
@@ -1,19 +1,11 @@
-## [0.0.4](https://github.com/technovangelist/airenamer/compare/v0.0.3...v0.0.4) (2024-01-06)
+## 0.0.6 (2024-01-14)
```

Comment:
remove changes to changelog. this is updated automatically

Reply:
Removed. Sorry for that but i do not have experience with github actions since i use GitLab CI/CD.